Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -164,6 +164,63 @@ function vtkMouseRangeManipulator(publicAPI, model) {
     const size = glRenderWindow.getViewportSize(renderer);
     // rescale size to match mouse event position
     model.containerSize = size.map((v) => v * ratio);
+
+    if (model.usePointerLock && !interactor.isPointerLocked()) {
+      interactor.requestPointerLock();
+      publicAPI.startPointerLockInteraction(interactor);
+    }
+  };
+
+  publicAPI.onButtonUp = () => {
+    publicAPI.exitPointerLock();
+  };
+
+  publicAPI.bindEvents = (container) => {
+    document.addEventListener(
```

Comment:
This is already handled in the [RenderWindowInteractor bindEvents](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/RenderWindowInteractor/index.js#L243), and so it shouldn't be necessary here.

Reply:
It should be good now